app/controllers/users_controller.rb

```diff
@@ -14,18 +23,57 @@ def create
       flash[:success] = "Welcome to the Sample App!"
       redirect_to @user
     else
-      flash[:danger] = "Danger! Please try again."
+      flash.now[:danger] = "Sorry. Please try again."
       render :new
     end
   end
 
   def edit
-    @user = User.find params[:id]
+    @user = User.find_by id: params[:id]
```

Comment:
```suggestion

```

Reply:
Do you mean I need to delete the line32? or change to anything? I do not see what's the different between mine and suggested change